builder/builder.go

```diff
@@ -1737,13 +1741,47 @@ func (b *Builder) BuildImage(format string, template string) error {
 		format = b.State.Mix.Format
 	}
 
-	// If the user did not pass in a template, default to release-image-config.json
+	// swupd (client) called by itser will need a temporary directory to act as its stage dir.
+	wd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	// If the user did not pass in a template, use the default
 	if template == "" {
-		template = "release-image-config.json"
+		template = imageTemplate
+
+		// If  the default image template is not present in the mix workspace,
+		// copy it from the default ister config directory and update the bundle list based on mix bundles
+		templateFile := filepath.Join(wd, template)
+		if _, err = os.Stat(templateFile); os.IsNotExist(err) {
+			fmt.Printf("Warning: Image template %s not found\n", templateFile)
+
+			configFile := filepath.Join(isterConfigDir, template)
+			fmt.Printf("Copying image template from %s\n", configFile)
+
+			// Read ister template
+			var configValues []byte
+			configValues, err = ioutil.ReadFile(configFile)
+			if err != nil {
+				return err
+			}
+
+			mixBundlesFile := filepath.Join(wd, b.MixBundlesFile)
```

Comment:
This will absolutely fail if running mixer from outside the mixer working directory.

Reply:
Done. Using b.Config.Builder.VersionPath to obtain mixer working directory